### ISSUE TITLE
fulldb Error in processCsv on row 1

### ISSUE THREAD

**Issue by zdavatz** [zdavatz/aips2sqlite#14]
running on Production
  
`java -jar -Xmx6000m aips2sqlite.jar --lang=de --verbose --zurrose=fulldb`

results in

```
Saved encrypted file rose_access.ami.ser
Processing BAG preparations file... 6010
>> Error in processCsv on row 1
```

**Comment by zdavatz** [comment]
to build on Windows, see #15

**Comment by zdavatz** [comment]
fixed with: 869d1a0a0266a672140119f235ebe8d22b681727
